### ISSUE TITLE
[Accessibility][PDF Viewer] PDF Viewer headings are not announced  

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1748]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Using arrow keys, attempt to read PDF headings (e.g. "Bodea Brochure").

**Current Behavior**
Visible heading text for PDF documents in the PDF Viewer header (e.g. "Bodea Brochure") are hidden from assistive technology.

This content is marked up as a navigation element, with the visible text given a role of link and an aria-current attribute attribute identifying the link as the current page, but the entire component is hidden from assistive technology with aria-hidden="true" on a parent element. Screen reader users do not have access to the text identifying the PDF. 

**Expected behavior/code**
Visible heading text is announced and provides appropriate semantic information.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users

**WCAG SC**
1.3.1 Info and Relationships

**Code snippet**
```
<div class="sdk-HeaderFileInfoView-headerNoBranding" aria-hidden="true">
    <span></span>
    <span class="sdk-HeaderFileInfoView-breadcrumbs">
        <nav aria-label="Breadcrumbs">
            <ul class="spectrum-Breadcrumbs">
                <li class="spectrum-Breadcrumbs-item">
                    <div class="spectrum-Breadcrumbs-itemLink" role="link" aria-current="page">
                        <span class="sdk-HeaderFileInfoView-fileBreadCrumb sdk-HeaderFileInfoView-fileBreadCrumbNormal">Bodea Brochure</span>
                    </div>
                </li>
            </ul>
        </nav>
    </span>
    <span class="sdk-HeaderFileInfoView-statusBlock sdk-HeaderFileInfoView-displayNone">
        <button class="spectrum-Button spectrum-Button--primary is-disabled sdk-HeaderFileInfoView-save sdk-HeaderFileInfoView-displayNone" disabled=""><span>Save</span></button>
    </span>
</div>
```

**Possible Solution**
Consider whether the aria-hidden attribute can be removed from the parent element of this text. If this element does not behave like a breadcrumb, consider whether it can be replaced with appropriate heading markup without the navigation and link semantics.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
